spec/compiler/semantic/automatic_cast_spec.cr

```diff
@@ -208,7 +208,7 @@ describe "Semantic: automatic cast" do
 
       foo(:four)
       ),
-      "expected argument #1 to 'foo' to be Foo, not Symbol"
+      "expected argument #1 symbol to 'foo' to match a Foo enum member"
```

Comment:
I find the wording "argument #1 symbol" a bit confusing. Can we keep the structure "argument #1 to 'foo'"?
Maybe the full sentence could be something like "Expected argument #1 to 'foo' to match a member of enum Foo"?
This does not explicitly mention symbol autocasting, not sure if that's necessary? We could add "... or a symbol that autocasts". But I think it's actually okay without that. The message implicitly acknowledges that the symbol argument is tried to be autocasted but wouldn't match.

Reply:
Done!